src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/CommentScenarios.java

```diff
@@ -247,8 +247,17 @@ public void shouldReturn200WhenDeleteCommentById() {
         final String commentId = UUID.randomUUID().toString();
         final ValidatableResponse response = createComment(annotationId, commentId);
         final String id = extractJSONObjectFromResponse(response).getString("id");
+        final ValidatableResponse deletedResponse = deleteCommentById(id);
 
-        deleteCommentById(id).statusCode(200);
+        deletedResponse.statusCode(200);
+    }
+
+    @Test
+    public void shouldReturn500WhenDeleteCommentById() {
+        final String commentId = UUID.randomUUID().toString();
+        final ValidatableResponse deletedResponse = deleteCommentById(commentId);
+
+        deletedResponse.statusCode(500);
```

Comment:
why is it supposed to return 500?

Reply:
at the moment api returning 500 when given id is not found and raised a bug for this issue. API should return 404 when id not found. For now added this test case and when the bug is fixed this test case will fail, so we need to update this test case to make it to pass.